modules/core/src/main/scala/gem/math/Declination.scala

```diff
@@ -0,0 +1,101 @@
+  assert(
+    toAngle.toMicroarcseconds >= Angle.Angle270.toMicroarcseconds ||
+    toAngle.toMicroarcseconds <= Angle.Angle90.toMicroarcseconds,
+    s"Invariant violated. $toAngle is outside the range [270 - 360) + [0 - 90]"
+  )
+
+  /**
+   * Offset this [[Declination]] by the given angle, returning the result and a carry bit. A carry
+   * of `true` indicates that the result lies on the opposite side of the sphere and the
+   * associated [[RightAscension]] (if any) must be flipped by around the 90° axis. Exact,
+   * invertible by offseting again by `-a` if carry is false, or by `a` if true; new carry will be
+   * the same.
+   * @group Operations
+   */
+  def offset(a: Angle): (Declination, Boolean) =
+    Declination.fromAngleWithCarry(toAngle + a)
+
+  // override to show signed angle
+  final override def toString: String = {
+    val dms = toAngle.toDMS; import dms._
+    val signedDegrees = if (degrees > 180) degrees - 360 else degrees
+    f"Dec($signedDegrees:$arcminutes%02d:$arcseconds%02d.$milliarcseconds%03d$microarcseconds%03d)"
+  }
+
+  final override def equals(a: Any) =
+    a match {
+      case dec: Declination => dec.toAngle === this.toAngle
+      case _                => false
+    }
+
+  final override def hashCode =
+    toAngle.hashCode
+
+}
+
+object Declination {
+
+  val DeclinationMin:  Declination = unsafeFromAngle(Angle.Angle270)
+  val DeclinationMax:  Declination = unsafeFromAngle(Angle.Angle90)
+  val DeclinationZero: Declination = unsafeFromAngle(Angle.Angle0)
+
+  /**
+   * Construct a `Declination` from an `Angle` in [270 - 360) + [0 - 90], if possible.
+   * @group Constructors
+   */
+  def fromAngle(a: Angle): Option[Declination] =
+    if (
+      a.toMicroarcseconds >= Angle.Angle270.toMicroarcseconds ||
+      a.toMicroarcseconds <= Angle.Angle90.toMicroarcseconds
+    ) Some(new Declination(a) {}) else None
+
+  /**
+   * Construct a `Declination` from an `Angle`, mirroring about the 90° axis if out of range and
+   * reporting whether or not mirroring was required. This operation is useful when offsetting might
+   * cause coordinates to cross the pole, in which case the associated RA will need to be flipped
+   * 180°.
+   * @group Constructors
+   */
+  def fromAngleWithCarry(a: Angle): (Declination, Boolean) =
+    Declination.fromAngle(a).map((_, false)) getOrElse {
+      (Declination.unsafeFromAngle(a mirrorBy Angle.Angle90), true)
+    }
+
+  /**
+   * Construct a `Declination` from an `Angle` in [270 - 360) + [0 - 90], raising an exception if
+   * out of range.
+   * @group Constructors
+   */
+  def unsafeFromAngle(a: Angle): Declination =
+    fromAngle(a).getOrElse(sys.error(s"Declination out of range: $a"))
+
+  /**
+   * Declinations are ordered from north to south.
+   * @group Typeclass Instances
+   */
+  implicit val DeclinationOrder: Order[Declination] =
+    Order.orderBy(_.toAngle.toSignedMicroarcseconds)
```

Comment:
Sorry, I think you're right.  I missed the `Signed` part of `toSignedMicroarcseconds`.  The comment seems wrong then?  `Declinations are ordered from north to south.`